validations.go

```diff
@@ -30,6 +32,21 @@ type validateCreateable interface {
 }
 
 func (m *Model) validateCreate(c *Connection) (*validate.Errors, error) {
+	v := reflect.Indirect(reflect.ValueOf(m.Value))
+	if v.Kind() == reflect.Slice || v.Kind() == reflect.Array {
+		var err error
+		for i := 0; i < v.Len(); i++ {
+			val := v.Index(i)
+			newModel := &Model{Value: val.Addr().Interface()}
+			verrs, err := newModel.validateCreate(c)
+
+			if err != nil || verrs.HasAny() {
```

Comment:
I like this pattern, can you clean up the other methods that have them broken out onto separate lines?